### PR TITLE
Fix vulnerability in uploads service

### PR DESCRIPTION
**Pull request by mojotalantikite**
## Summary

This pulls in the changes from the quickstart repository to mitigate against potential injection attacks in the `uploads` service.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2499

### CODE REVIEW

File: services/uploads/src/antivirus.js [hunk 1]
```diff
@@ -17,6 +18,11 @@ const constants = require('./constants');
  * @return {int} Length of S3 object in bytes.
  */
 async function sizeOf(key, bucket) {
+    console.log('key: ' + key);
+    console.log('bucket: ' + bucket);
+    var sts = new AWS.STS();
```
Comment (macrael) [inline services/uploads/src/antivirus.js:23]: are either of these two varaibles used?

File: services/uploads/src/antivirus.js [hunk 2]
```diff
@@ -33,25 +39,18 @@ async function isS3FileTooBig(s3ObjectKey, s3ObjectBucket) {
...
-    const downloadDir = `/tmp/download`;
-    if (!fs.existsSync(downloadDir)) {
-        fs.mkdirSync(downloadDir);
+    if (!fs.existsSync(constants.TMP_DOWNLOAD_PATH)) {
+        fs.mkdirSync(constants.TMP_DOWNLOAD_PATH);
     }
 
-    let localPath = pathFromObjectKey(s3ObjectKey);
+    const tmpFileName = `${crypto.randomUUID()}.tmp`;
+    let localPath = `${constants.TMP_DOWNLOAD_PATH}${tmpFileName}`;
```
Comment (macrael) [inline services/uploads/src/antivirus.js:53]: ugh, i should have reviewed this in their PR. I wish this was actually shared code!! We should be using path.join here.

**Review by macrael — APPROVED** [review]
Looks good. Let’s just keep the code like it is upstream if either of my comments conflict with that.

File: services/uploads/src/antivirus.js (hunk 1 above)
Comment (mojotalantikite) [reply services/uploads/src/antivirus.js:23]: They don't seem to be -- I just copied the files over, but yeah I think we can delete these.

File: services/uploads/src/antivirus.js (hunk 2 above)
Comment (mojotalantikite) [reply services/uploads/src/antivirus.js:48]: yup good catch, I'll fix it

**Comment by JasonLin0991** [comment]
@mojotalantikite I believe you're cypress test is failing because the `rate-certification-programs` was turned on. You should be able to get this passing if we toggle it off.

**Comment by haworku** [comment]
@mojotalantikite sorry that my bad I forgot to toggled off the flag while I was testing like @JasonLin0991 said. also fyi for the cypress github action I'm bumping that on the cypress dependabot branch now after running into an issue

**Pull request merged**